src/components/BounceDetailsContainer/index.jsx

```diff
@@ -92,14 +94,26 @@ const BounceRuleDetailed = ({
           <Column className="details-button-column" width={1} offset={11}>
             <span>
               <Button
-                onClick={handleEditClicked}
+                onClick={
+                  editText === "FREE" || editText === "ALREADY"
+                    ? handleEditClicked
```

Comment:
Let's wrap this in a function to avoid assigning to two different functions onClick={handleConcurrentEditClicked} i.e. ```const handleConcurrentEditClicked = (e) => { if (userCanEditRule) { handleEditClicked(e); } }```